bin/aws_cleaner.rb

```diff
@@ -26,84 +27,131 @@
 require_relative '../lib/aws-cleaner.rb'
 
 def config(file)
-  YAML.load(File.read(file))
+  YAML.safe_load(File.read(File.expand_path(file)), [Symbol])
 rescue StandardError => e
   raise "Failed to open config file: #{e}"
 end
 
+def logger(config)
+  file = config[:log][:file] unless config[:log].nil?
+
+  if file
+    begin
+      # Check if specified file can be written
+      awslog = File.open(File.expand_path(file), File::CREAT | File::WRONLY | File::APPEND)
+      awslog.sync = true if awslog
+    rescue StandardError => e
+      $stderr.puts "aws-cleaner: ERROR - Failed to open log file #{file} beause of #{e}. STDOUT will be used instead."
+    end
+  else
+    $stdout.puts 'aws-cleaner: WARN - Log file is not specified. STDOUT will be used instead.'
+  end
+
+  # Use STDOUT if it is not possible to write to log file
+  awslog = STDOUT if awslog.nil?
```

Comment:
We will need to enable `sync` on STDOUT otherwise logs won't get flushed immediately. Something like

```
if awslog.nil?
  awslog = STDOUT
  awslog.sync = true
end
```